report/report.py

```diff
@@ -54,29 +45,53 @@ async def cmd_report(
         # Pre-emptively delete the message for privacy reasons
         await ctx.message.delete()
 
-        author = ctx.author
-        if author.bot:
```

Comment:
Are we no longer ignoring the bot's report?

Reply:
Bots don't respond to other bots. This functionality is baked into the commands module